pkg/modules/observability.go

```diff
@@ -284,21 +267,19 @@ func ObservabilityTopology(ctx context.Context, isDeleting bool, op utils.Operat
 	return nil
 }
 
-// getTopology - get topology yaml string
-func getTopology(op utils.OperatorConfig, cr csmv1.ContainerStorageModule) (string, error) {
-	YamlString := ""
-
+func getTopology(op utils.OperatorConfig, cr csmv1.ContainerStorageModule) ([]crclient.Object, error) {
 	obs, err := getObservabilityModule(cr)
 	if err != nil {
-		return YamlString, err
+		return nil, err
 	}
 
 	buf, err := readConfigFile(obs, cr, op, TopologyYamlFile)
 	if err != nil {
-		return YamlString, err
+		return nil, err
 	}
-	YamlString = string(buf)
+	YamlString := string(buf)
 
+	fmt.Println(string(buf))
```

Comment:
Do we need this `println `statement here?

Reply:
Good observation. Removed it.